packages/atomic/src/components/result-template-components/atomic-result-printable-uri/atomic-result-printable-uri.tsx

```diff
@@ -0,0 +1,110 @@
+import {Component, h, Prop, State} from '@stencil/core';
+import {Result} from '@coveo/headless';
+import {ResultContext} from '../result-template-decorators';
+import {parseXML} from '../../../utils/utils';
+import {
+  Bindings,
+  BindStateToI18n,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+
+/**
+ * The ResultUri component displays the URI, or path, to access a result.
+ * @part result-printable-uri-list - The result printable uri list
+ * @part result-printable-uri-list-expanded - The expanded result printable uri list
+ * @part result-printable-uri-list-element - A result printable uri list element
+ * @part result-printable-uri-link - A result printable uri clickable link
+ * @part result-printable-uri-list-ellipsis - The clickable ellipsis of a result uri
+ */
+@Component({
+  tag: 'atomic-result-printable-uri',
+  styleUrl: 'atomic-result-printable-uri.pcss',
+  shadow: false,
+})
+export class AtomicResultPrintableUri {
+  @InitializeBindings() public bindings!: Bindings;
+  @ResultContext() private result!: Result;
+  private minNumOfParts = 3;
+
+  @State() listExpanded = false;
+  @BindStateToI18n()
+  @State()
+  private strings = {
+    collapsedUriParts: () => this.bindings.i18n.t('collapsedUriParts'),
+  };
+  @State() error!: Error;
+
+  /**
+   * The maximum number of Uri parts to display, has to be over the minimum of `3` in order to be effective. Putting `0` will disable the ellipsis.
+   */
+  @Prop() maxNumOfParts = 5;
```

Comment:
Could we add a validation on `maxNumOfParts` that shows an error when a value less than 3 is set?

For disabling ellipses, could we use `Infinity` instead of 0? Does stencil parse it correctly as a number?
If so, I think this would be clearer compared to using values like `0` or `-1`. If not, then we can keep `0`,